docs/source/get_started/installation.rst

```diff
@@ -167,7 +167,7 @@ If you are using the Windows and the PowerShell:
 
 
 .. _`DeepChem Tutorials`: https://github.com/deepchem/deepchem/tree/master/examples/tutorials
-.. _`forum post`: https://forum.deepchem.io/t/getting-deepchem-running-in-colab/81
+.. _`forum post`: https://forum.deepchem.io/t/getting-deepchem-running-in-colab/81/7?u=nd-02110114
```

Comment:
I think the copy paste here added in your user id which should be removed

Reply:
Yes, this is a mistake. I fixed.